docs/command/atlas-api-teams-getProjectTeam.txt

```diff
@@ -0,0 +1,104 @@
+.. _atlas-api-teams-getProjectTeam:
+
+==============================
+atlas api teams getProjectTeam
+==============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+`Public Preview: please provide feedback <https://feedback.mongodb.com/forums/930808-atlas-cli>`_: Returns one team to which the authenticated user has access in the project specified using its unique 24-hexadecimal digit identifier.
+
+Public Preview: The atlas api sub-command, automatically generated from the MongoDB Atlas Admin API, offers full coverage of the Admin API and is currently in Public Preview (please provide feedback at https://feedback.mongodb.com/forums/930808-atlas-cli).
+Admin API capabilities have their own release lifecycle, which you can check via the provided API endpoint documentation link.
+
+
+
+All members of the team share the same project access. To use this resource, the requesting Service Account or API Key must have the Project Read Only role.
+
+This command is autogenerated and corresponds 1:1 with the Atlas API endpoint https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Teams/operation/getProjectTeam.
+
+For more information and examples, see the referenced API documentation linked above.
+
+Syntax
+------
+
+.. code-block::
+   :caption: Command Syntax
+
+   atlas api teams getProjectTeam [options]
+
+.. Code end marker, please don't delete this comment
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - --envelope
+     - envelope
+     - false
+     - Flag that indicates whether Application wraps the response in an envelope JSON object. Some API clients cannot access the HTTP response headers or status code. To remediate this, set envelope=true in the query. Endpoints that return a list of results use the results object as an envelope. Application adds the status parameter to the response body.
+   * - --groupId
+     - string
+     - true
+     - Unique 24-hexadecimal digit string that identifies your project. Use the [/groups](#tag/Projects/operation/listProjects) endpoint to retrieve all projects to which the authenticated user has access.
+       
+       **NOTE**: Groups and projects are synonymous terms. Your group id is the same as your project id. For existing groups, your group/project id remains the same. The resource and corresponding endpoints use the term groups.
+   * - -h, --help
+     - 
+     - false
+     - help for getProjectTeam
+   * - -o, --output
+     - string
+     - false
+     - preferred api format, can be ["json", go-template] This value defaults to "json".
```

Comment:
```suggestion
     - Preferred API format, can be ["json", go-template] This value defaults to "json".
```
We need to be consistent about capitalization for the opening word. Throughout the CLI docs we have favored capitalizing these, so let's stick to that. As for api -> API, "api" should only be used if we're referencing some specific syntax that uses it in lowercase, otherwise it's an acronym and should be capitalized appropriately.